### PR TITLE
Kaleem neslit.9244.copy wallet address

### PR DESCRIPTION
**Pull request by KaleemNeslit**
## Link to Issue
Closes: #9244 

## Description of Changes
- Created the new component  CWSingleSelectItem and pass it it to React select to show icon on selected address
- 

## Test Plan
N/A
  - 

## Deployment Plan
N/A

## Other Considerations

-

### CODE REVIEW

File: packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWSingleSelectItem/CWSingleSelectItem.tsx
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWSingleSelectItem/CWSingleSelectItem.tsx:30]: name of the class should be the same as name of the component

File: packages/commonwealth/client/scripts/views/modals/ManageCommunityStakeModal/StakeExchangeForm/StakeExchangeForm.tsx
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/modals/ManageCommunityStakeModal/StakeExchangeForm/StakeExchangeForm.tsx:310]: please use `saveToClipboard` util we have in the codebase. It also supports notification toast

File: packages/commonwealth/client/scripts/views/modals/ManageCommunityStakeModal/StakeExchangeForm/StakeExchangeForm.tsx
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/modals/ManageCommunityStakeModal/StakeExchangeForm/StakeExchangeForm.tsx:138]: is this really necessary?

File: packages/commonwealth/client/scripts/views/modals/ManageCommunityStakeModal/StakeExchangeForm/StakeExchangeForm.tsx
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/modals/ManageCommunityStakeModal/StakeExchangeForm/StakeExchangeForm.tsx:348]: just `showIcon` is sufficient

**Review by masvelio — APPROVED** [review]
approving in advance as it works fine, please see my comments

File: packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWSingleSelectItem/CWSingleSelectItem.tsx
Comment (mzparacha) [inline packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWSingleSelectItem/CWSingleSelectItem.tsx:8]: `extraProp` as a prop name is too vague, lets rename to something specific

File: packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWSingleSelectItem/CWSingleSelectItem.tsx
Comment (mzparacha) [inline packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWSingleSelectItem/CWSingleSelectItem.tsx:9]:
```diff
- handleClickCopyClipboard?: (id: string) => void;
+ handleClickCopyClipboard?: (text: string) => void;
```

File: packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWSelectList/CWSelectList.tsx
Comment (mzparacha) [inline packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWSelectList/CWSelectList.tsx:18]:
```diff
- handleClickCopyClipboard?: (id: string) => void;
+ handleClickCopyClipboard?: (text: string) => void;
```

File: packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWSelectList/CWSelectList.tsx
Comment (mzparacha) [inline packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWSelectList/CWSelectList.tsx:19]: `showIcon` is too vague, lets rename to `showCopyIcon` or something more specific

File: packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWSelectList/CWSelectList.tsx
Comment (mzparacha) [inline packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWSelectList/CWSelectList.tsx:139]: this should only be rendered when we want to show copy icon, lets add this conditionally

File: packages/commonwealth/client/scripts/views/modals/ManageCommunityStakeModal/StakeExchangeForm/StakeExchangeForm.tsx
Comment (KaleemNeslit) [reply packages/commonwealth/client/scripts/views/modals/ManageCommunityStakeModal/StakeExchangeForm/StakeExchangeForm.tsx:138]: yes test are falling because of that even i didn't find anything wrong with code

**Review by Israellund — APPROVED** [review]
Approving this now as well since it works, but please make sure all requested changes are made and conversations are resolved before merging.

**Pull request merged**
